### ISSUE TITLE
pCloud Support

### ISSUE THREAD

**Issue by syepes** [gilbertchen/duplicacy#295]
This tool look very interesting but it's missing my storage solution, pCloud have a great API https://docs.pcloud.com hope this feature gets come traction

**Comment by tophee** [comment]
Indeed! And pcloud supports webdav, so pcloud could also be supported by [supporting webdav](https://github.com/gilbertchen/duplicacy/issues/4). Any plans?

**Comment by sergeevabc** [comment]
I guess using syncing local backup with pCloud via Rclone might be considered a workaround for now?

**Comment by tophee** [comment]
You don't need rclone. duplicacy can access pcloud directly via webDAV.

If you have any further questions, please use the new duplicacy forum at https://forum.duplicati.com

**Comment by tophee** [comment]
Thanks for your watchful eye! I typed it on my phone and used auto-complete for the url. I guess I should delete that one from my autocomplete dictionary.
